templates/music/album_detail.html

```diff
@@ -42,78 +43,89 @@
 {% block content %}
     <h3>{{ album.name }}</h3>
     <h4>BY {{ album.project.artist.name }}</h4>
-    <ul class="tabs" data-tabs id="music-tabs">
-        <li class="tabs-title is-active"><a href="#music-overview" aria-selected="true"><i class="fa fa-user fa-2x tab-icon" aria-hidden="true"></i><span class="artist-tab">Overview</span></a></li>
-        <li class="tabs-title"><a href="#music-stats"><i class="fa fa-money fa-2x tab-icon" aria-hidden="true"></i><span class="artist-tab">Stats</span></a></li>
-    </ul>
-    <div class="tabs-content" data-tabs-content="music-tabs">
-        {% include "music/includes/music_detail_overview.html" %}
-        {% include "music/includes/music_detail_stats.html" %}
+    <div class="project-image">
+        {% thumbnail album.artwork.img "250x250" as thumb %}
+            <img src="{{ thumb.url }}" width="{{ thumb.width }}" height="{{ thumb.height }}" alt="{{ album.name }}" />
+        {% endthumbnail %}
+    </div>
+
+    {% if user_is_investor %}
+        <div class="download-container">
+            <a class="button sign-up" href="{{ album.audio.get_temporary_url }}" download>
+                DOWNLOAD
+            </a>
+        </div>
+    {% else %}
+        <h5>You must be an investor in order to listen and download.</h5>
+    {% endif %}
+    <div class="break-small">
+        <hr>
+    </div>
+    <div>
+        <h4>Streams</h4>
+        <h4 class="color-blue">
+            {% for disc in album.discs %}
```

Comment:
Would it be better to display this sort of data in a table?

Reply:
Yeah, the challenge was that most of the releases are singles and the charts look funny with just one bar in them. I got feedback from a few people that they didn't look very good that way. 

I think the charts would look a lot better if we were able to display the data over time, but I wasn't sure how to do that so I thought leaving them out for now would be best.